README.md

```diff
@@ -344,3 +366,13 @@ like to apply them in Photoshop or any other code.
 | NeutralGrey           | 224 | 224 | 224 | \#E0E0E0 |
 | LightOchre            | 255 | 173 | 140 | \#FFAD8C |
 | UpdatedOchre          | 231 |  71 |  38 | \#E74726 |
+| Ochre                 | 231 |  71 |  38 | \#E74726 |
```

Comment:
Maybe have the RGB values and hex code as a text overlay on the swatch plot rather than a separate table down the bottom?

![swatchplot-1](https://github.com/user-attachments/assets/984673ff-6a45-46ec-bb63-cb3b0c8b6c68)

Reply:
Or I quite like how this site has it down below each palette in the main body:

<img width="605" alt="Screenshot 2024-08-28 at 8 53 48 AM" src="https://github.com/user-attachments/assets/b3e18463-12e7-4799-b397-205d9557bd1a">